spec/features/rent_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+feature 'tries to rent the car' do
+  let(:rent) { build(:rent) }
+  scenario 'they get alert with text "Loggade in"' do
+    visit '/bil/bokningar/ny'
+    find(:css, "#rent_disclaimer").set(true)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.